.github/workflows/ruby.yml

```diff
@@ -0,0 +1,30 @@
+# Run tests!
+name: "Ruby"
+
+on:
+  push: # Run on all pushes
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # NOTE: when updating the ruby versions, consider re-writing the rubocop_rules.yml
+        # and updating rules_version in spec/rubocop_rules_spec.rb
+        ruby-version: ['3.1']
```

Comment:
@sethherr I removed the other versions we had defined here during our pairing session as I found that it would try to set up three different CI jobs or (actions), two of which would always fail because of the `.ruby-version` file I have set for this repo. I'm not sure if that's a major concern or if I'm missing something on that front, but would love to pair if you think we should add back the other versions

Reply:
You should .gitignore the `.ruby-version` file in this repo.

Reply:
(after you gitignore the version, you can put the other versions back in here - we want it to run 3 different CI jobs)